src/main/java/slimeknights/tconstruct/tools/modifiers/ability/interaction/FirestarterModifier.java

```diff
@@ -161,4 +165,64 @@ public InteractionResult afterBlockUse(IToolStackView tool, int level, UseOnCont
     }
     return didIgnite ? InteractionResult.sidedSuccess(world.isClientSide) : InteractionResult.PASS;
   }
+
+  /** Ignites the given block */
+  private static boolean igniteDispenser(IToolStackView tool, Level world, BlockPos pos, BlockState state, Direction sideHit, Direction horizontalFacing) {
```

Comment:
Why did you duplicate this whole method just to remove a nullable player argument? Just call the original method by passing null.

If there are any other differences, add the needed parameter to distinguish the two